topics/mariadb/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: mariadb-server 
+created_by: Michael "Monty" Widenius, David Axmark, Allan Larsson
+display_name: MariaDB
+github_url: https://github.com/MariaDB/server
+logo: mariadb.png
+released: October 29, 2009
+short_description: MariaDB is a community-developed, open source relational database management system forked from MySQL.
+topic: mariadb
+url: https://mariadb.org/
+wikipedia_url: https://en.wikipedia.org/wiki/MariaDB
+---
+MariaDB is an open source relational database management system forked from the free MySQL database in 2009 by original MySQL developers. While maintaining high compatibility with MySQL, MariaDB adds new storage engines and features as a community-driven open source alternative. It has gained widespread adoption, being utilized by major companies and many Linux distributions.
```

Comment:
Nit: Newline at end of file please

Reply:
😭 newbie mistake. Fixed (as well as a trailing whitespace 😬)